Elsewhen/Helpers/DateHelpers.swift

```diff
@@ -129,37 +129,45 @@ func stringFor(time date: Date, in zone: TimeZone, sourceZone: TimeZone, ignorin
     if timeFormat == .twelve {
         result = result.lowercased().replacingOccurrences(of: "pm", with: pmSymbol).replacingOccurrences(of: "am", with: amSymbol)
     }
+    if lowercaseAMPM {
+        result = result.lowercased()
+    }
     return result
 }
 
-func stringForTimeAndFlag(in tz: TimeZone, date: Date, sourceZone: TimeZone, separator: MykeModeSeparator, timeZonesUsingEUFlag: Set<TimeZone>, timeZonesUsingNoFlag: Set<TimeZone>, showCities: Bool, ignoringTimeFormatOverride: Bool = false) -> String {
+func stringForTimeAndFlag(in tz: TimeZone, date: Date, sourceZone: TimeZone, separator: MykeModeSeparator, timeZonesUsingEUFlag: Set<TimeZone>, timeZonesUsingNoFlag: Set<TimeZone>, showCities: Bool, hideFlags: Bool = false, lowercaseAMPM: Bool = false, useShortNames: Bool = false, ignoringTimeFormatOverride: Bool = false) -> String {
     var text = ""
-    let abbr = tz.fudgedAbbreviation(for: date) ?? ""
-    let flag: String
-    if timeZonesUsingNoFlag.contains(tz) {
-        flag = NoFlagTimeZoneEmoji
+    let abbr = tz.fudgedAbbreviation(for: date, usingShort: useShortNames) ?? ""
+    var flag: String
```

Comment:
I would do the contortions to make this a `let`, but that's just my obsession with SSA form showing 😅

Reply:
I agree, but the contortions also annoy me 😆
Maybe in a future refactor...